### PR TITLE
add landing message. #1

### PR DESCRIPTION
**Pull request by AmauriMarcos**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
   </head>
   <body>
     <main>
-        <h1>Welcome to Pontify</h1>  
+        <h1>Welcome to Pontify ! Hope you enjoy your stay with us. </h1>
```
Comment (marquitos-santos) [inline index.html:13]: Could you remove 'enjoy your stay' string, fix the name Pontify to Portionfy and make 'Welcome to Portinfy' red ?

Comment (marquitos-santos) [reply index.html:13]: Sure, no problem.

Comment (AmauriMarcos) [reply index.html:13]: Sure @marquitos-santos , I can do it.

**Review by marquitos-santos — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by marquitos-santos — APPROVED** [review]

**Pull request merged**